src/navigation-helpers.js

```diff
@@ -0,0 +1,17 @@
+export const parseNavigationStructure = (pages) => {
+  if (pages === null) {
+    return null;
```

Comment:
Maybe it's safer to return an empty array here instead? :thinking:  

In case `parsedNavdata` is `null`, this line  `<SitemapList pages={parsedNavData} isTopLevel={true} />` would break when `SitemapList` tries to do `pages.map`